pkg/curatedpackages/packageclient.go

```diff
@@ -3,37 +3,50 @@ package curatedpackages
 import (
 	"fmt"
 	"os"
+	"path/filepath"
 	"strings"
 	"text/tabwriter"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+
 	packagesv1 "github.com/aws/eks-anywhere-packages/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/constants"
+	"github.com/aws/eks-anywhere/pkg/validations"
 )
 
 const (
-	minWidth = 16
-	tabWidth = 8
-	padding  = 0
-	padChar  = '\t'
-	flags    = 0
+	minWidth        = 16
+	tabWidth        = 8
+	padding         = 0
+	padChar         = '\t'
+	flags           = 0
+	CustomName      = "my-"
+	kind            = "Package"
+	filePermission  = 0o644
+	dirPermission   = 0o755
```

Comment:
```suggestion
	filePermission  = 0o600
	dirPermission   = 0o700
```
Better safe than sorry.